public/main/admin/export_exercise_results.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+declare(strict_types=1);
+
+use Chamilo\CoreBundle\Framework\Container;
+
+$cidReset = true;
+require_once __DIR__ . '/../inc/global.inc.php';
+
+api_protect_admin_script(true);
+
+// Setting the section (for the tabs).
+$this_section = SECTION_PLATFORM_ADMIN;
+
+$interbreadcrumb[] = ['url' => 'index.php', 'name' => get_lang('Administration')];
+
+$toolName = get_lang('Export all exercise results');
+
+// Get request parameters
+$sessionId = isset($_REQUEST['session_id']) ? (int) $_REQUEST['session_id'] : null;
+$courseId = isset($_GET['selected_course']) ? (int) $_GET['selected_course'] : null;
+$exerciseId = isset($_REQUEST['exerciseId']) ? (int) $_REQUEST['exerciseId'] : null;
+
+// Get repositories
+$user = api_get_user_entity(api_get_user_id());
+$sessionRepository = Container::getSessionRepository();
+$courseRepository = Container::getCourseRepository();
+$cQuizRepository = Container::getCQuizRepository();
```

Comment:
Container already exposes getQuizRepository(): CQuizRepository. Adding getCQuizRepository() introduces redundant naming for the same service.